src/networkcloud/azext_networkcloud/aaz/latest/networkcloud/storageappliance/_enable_remote_vendor_management.py

```diff
@@ -0,0 +1,331 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "networkcloud storageappliance enable-remote-vendor-management",
+    is_experimental=True,
+)
+class EnableRemoteVendorManagement(AAZCommand):
+    """Enable remote vendor management of the provided storage appliance.
+
+    :example: Turn on remote vendor management for storage appliance
+        az networkcloud storageappliance enable-remote-vendor-management --resource-group "resourceGroupName" --storage-appliance-name "storageApplianceName"
+    """
+
+    _aaz_info = {
+        "version": "2022-12-12-preview",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.networkcloud/storageappliances/{}/enableremotevendormanagement", "2022-12-12-preview"],
+        ]
+    }
+
+    AZ_SUPPORT_NO_WAIT = True
+
+    def _handler(self, command_args):
+        super()._handler(command_args)
+        return self.build_lro_poller(self._execute_operations, self._output)
+
+    _args_schema = None
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            required=True,
+        )
+        _args_schema.storage_appliance_name = AAZStrArg(
+            options=["-n", "--name", "--storage-appliance-name"],
+            help="The name of the storage appliance.",
+            required=True,
+            id_part="name",
+            fmt=AAZStrArgFormat(
+                pattern="^([a-zA-Z0-9][a-zA-Z0-9-_]{0,28}[a-zA-Z0-9])$",
+            ),
+        )
+
+        # define Arg Group "StorageApplianceEnableRemoteVendorManagementParameters"
+
+        _args_schema = cls._args_schema
+        _args_schema.support_endpoints = AAZListArg(
```

Comment:
You can hidden this argument in aaz-dev-tools if it's deprecated

Reply:
thanks! we didn't know there is a way to hide it

Reply:
NP, here's the doc [link](https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#hidden-arguments)